README.md

```diff
@@ -35,17 +35,38 @@ After cloning for the first time, change your app identifier inside
 
 ```jsonc
 {
-  // ...
-  // The default "com.tauri.dev" will prevent you from building in release mode
-  "identifier": "com.my-application-name.app",
-  // ...
+    // ...
+    // The default "com.tauri.dev" will prevent you from building in release mode
+    "identifier": "com.my-application-name.app"
+    // ...
 }
 ```
 
+Install pnpm:
+
+```shell
+npm install -g pnpm
```

Comment:
Would not recommend telling users a default way of installing pnpm since there are many ways of doing it depending on the operating system (e.g. I use brew on macOS and winget on Windows for pnpm).